demo/demo-edge/consumer/src/main/java/org/apache/servicecomb/demo/edge/consumer/Consumer.java

```diff
@@ -111,6 +114,33 @@ public void run(String prefix) {
     checkResult("v2/dec", decV2Result, "2.0.0");
   }
 
+  public void testEncrypt() {
+    prepareEdge("encryptApi");
+    String url = edgePrefix + "/v2/encrypt";
+
+    HttpHeaders headers = new HttpHeaders();
+    headers.setContentType(MediaType.MULTIPART_FORM_DATA);
+
+    MultiValueMap<String, String> form = new LinkedMultiValueMap<>();
+    form.add("name", "userName");
+    form.add("age", "10");
+    form.add("serviceToken", "serviceTokenTest");
+    form.add("hcrId", "hcrIdTest");
+    form.add("body", "bodyKey-hcrIdTest-{\"body1\":\"b1\",\"body2\":\"b2\",\"body3\":\"b3\"}");
+
+    HttpEntity<MultiValueMap<String, String>> entity = new HttpEntity<>(form, headers);
+
+    @SuppressWarnings("unchecked")
+    Map<String, Object> result = (Map<String, Object>) template.postForObject(url, entity, Map.class);
+    Assert.isTrue(result.containsKey("signature"), "must exist signature");
+    result.remove("signature");
+
+    String expected = "{name=userName, age=10, userId=serviceTokenTest-userId, body1=b1, body2=b2, body3=b3}";
+    System.out.println("encrypt result: " + result.toString());
```

Comment:
We are not supposed to use System.out.println in the test.

Reply:
done